samples/apps/copilot-chat-app/webapi/CopilotChat/Models/ChatMessage.cs

```diff
@@ -150,7 +151,14 @@ public static ChatMessage CreateBotResponseMessage(string chatId, string content
     /// <returns>A formatted string</returns>
     public string ToFormattedString()
     {
-        return $"[{this.Timestamp.ToString("G", CultureInfo.CurrentCulture)}] {this.UserName}: {this.Content}";
+        var content = this.Content;
```

Comment:
[nit] Remove this and just use 'this.Content' everywhere.

Reply:
synced offline, leaving as-is as this is by design